docs/custom_html_writer.py

```diff
@@ -33,7 +33,7 @@
 """
 
 
-class CustomHTMLWriter(html.SmartyPantsHTMLTranslator):
+class CustomHTMLWriter(html.HTML5Translator):
```

Comment:
@tswast I had to fix https://github.com/dhermes/bezier/blob/master/docs/custom_html_writer.py when Smarty Pants was removed. Any reason to pick `HTML5Translator` over `HTMLTranslator`?

Reply:
Nope. Just the first one I saw. I can change to HTMLTranslator.